### PR TITLE
Crypto currency status

### PR DESCRIPTION
**Pull request by yodit93**
### In this branch, I

- [ ] added a Home page with the list of crypto coins with their specific rank,
- [ ] added the forward link on each list,
- [ ] created a route for icons and made the application navigate to the details page in response to user clicks or taps, 
- [ ] retrieved data from the API to show detailed information about the corresponding coin
- [ ] created a route for the "Back" button (<), to navigate the user to the home page in response to clicking or tapping,
- [ ] added unit and integration tests,

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  box-sizing: border-box;
+}
+
+body {
+  background-color: orangered;
+}
+
+.header {
+  height: 30vh;
+  width: 100%;
```
Comment (DuaneDave) [inline src/index.css]: Good job on your project, it looks awesome. However there is one thing you have overlooked, which is the correct font family  (Lato and gill sans) for the website. Please use the fonts as specified in the project descriptions in order to make your design match with the template

File: src/__test__/Details.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        },
+      ],
+      isLoading: false,
+      error: null,
+    },
+  });
+  store.dispatch = jest.fn();
+  useParams.mockReturnValue({
+    id: '1',
+  });
```
Comment (DuaneDave) [inline src/__test__/Details.test.js]: [OPTIONAL] All the tests are passing, but I encourage you to make sure that the mocked data are still to an extent match with the one currently used in the application for consistency purposes.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @yodit93 👋, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

You can also consider:
- Making the presentation match with the new project setup with is crypto
- Making your documentation reflect the new topic you have chosen which is crypto for consistency purposes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/__test__/Details.test.js (hunk 2 above)
Comment (yodit93) [reply src/__test__/Details.test.js]: Ok, I will correct it. Thank you!!

File: src/index.css (hunk 1 above)
Comment (yodit93) [reply src/index.css]: Thanks
I will update it.

**Review by cynthiainga — APPROVED** [review]
Hi @yodit93, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] The app work correctly ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**